scenarios/sumo/platoon/merge_exit_sumo_t_agents_1/scenario.py

```diff
@@ -10,15 +10,14 @@
     Flow,
     JunctionModel,
     MapSpec,
-    Mission,
     Route,
     Scenario,
-    ScenarioMetadata,
     Traffic,
     TrafficActor,
     TrapEntryTactic,
     Trip,
 )
+from smarts.sstudio.types.scenario import StandardMetadata
```

Comment:
Minor, but makes sense to import it from `smarts.sstudio.types` above with the other imports.